cli/command/container/formatter_stats.go

```diff
@@ -181,45 +181,45 @@ func (c *statsContext) ID() string {
 
 func (c *statsContext) CPUPerc() string {
 	if c.s.IsInvalid {
-		return fmt.Sprintf("--")
+		return "--"
 	}
 	return fmt.Sprintf("%.2f%%", c.s.CPUPercentage)
 }
 
 func (c *statsContext) MemUsage() string {
 	if c.s.IsInvalid {
-		return fmt.Sprintf("-- / --")
+		return "-- / --"
 	}
 	if c.os == winOSType {
 		return units.BytesSize(c.s.Memory)
 	}
-	return fmt.Sprintf("%s / %s", units.BytesSize(c.s.Memory), units.BytesSize(c.s.MemoryLimit))
```

Comment:
Nit: I'm not really enthousiast with this modification, even if I understand why. I think the sprintf version os easier to read and review. Anyway it's just nitpicking here.

Reply:
Let me do those separate, as they're not strictly needed for the linter, but given that this stats output is called in a loop (to refresh stats), I thought it would be worth to change them. Looking at the file, I now see some other ones that could be adjusted in that respect, so let me change these back and do it separately.

Reply:
oh! of course it helps if I ping you after that; updated 👍 PTAL

Reply:
Still LGTM 🤗